src/test/resources/adapters/icarol/agencies.json

```diff
@@ -0,0 +1,316 @@
+          "city": "CarpetHanger",
+          "stateProvince": "CA",
+          "country": "United States",
+          "zipPostalCode": "12345",
+          "type": "PhysicalLocation"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Mailing",
+          "line1": "12345 Cool Street",
+          "county": "-1",
+          "city": "CarpetHanger",
+          "stateProvince": "CA",
+          "country": "United States",
+          "zipPostalCode": "12345",
+          "type": "PostalAddress"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Unspecified",
+          "number": "123-465-7890",
+          "label": "CarpetHanger Office",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Unspecified",
+          "number": "234-567-8901",
+          "label": "English",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Unspecified",
+          "number": "345-678-9012",
+          "label": "Spanish",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Unspecified",
+          "number": "456-789-0123",
+          "label": "Vietnamese",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Unspecified",
+          "number": "567-890-1234",
+          "label": "Farsi",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "Fax",
+          "number": "678-901-2345",
+          "label": "Fax",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "purpose": "MYY",
+          "number": "789-012-3456",
+          "label": "MYY",
+          "type": "PhoneNumber"
+        },
+        "isConfidential": false
+      },
+      {
+        "contact": {
+          "label": "Main Contact",
+          "type": "Person"
+        },
+        "isConfidential": true
+      },
+      {
+        "contact": {
+          "url": "www.example.org",
+          "label": "Website",
+          "type": "Website"
+        },
+        "isConfidential": false
+      }
+    ],
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
```

Comment:
description should differ with description text to verify if we're mapping the proper field

Reply:
it was replaced with lorem ipsum only if description and description text were equal